draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -610,6 +626,12 @@ application. RTP retransmissions can be done in the same RTP session or a
 separate RTP session {{!RFC4588}} and the flow identifier MUST be set to the
 flow identifier of the RTP session in which the retransmission happens.
 
+# Connection Shutdown
+
+Both peers MAY close the connection for variety of reasons. If one of the peers
```

Comment:
I think this is "Either peer MAY close the connection for a variety of reasons."